Dockerfile

```diff
@@ -7,6 +7,7 @@ COPY --from=eclipse-temurin:8-jdk-jammy /opt/java/openjdk /usr/lib/jvm/8
 COPY --from=eclipse-temurin:11-jdk-jammy /opt/java/openjdk /usr/lib/jvm/11
 COPY --from=eclipse-temurin:17-jdk-jammy /opt/java/openjdk /usr/lib/jvm/17
 COPY --from=eclipse-temurin:21-jdk-jammy /opt/java/openjdk /usr/lib/jvm/21
+COPY --from=openjdk:24-jdk /usr/java/openjdk-24 /usr/lib/jvm/24
```

Comment:
`openjdk` is deprecated: https://hub.docker.com/_/openjdk
You would have to use `eclipse-temurin:24-jdk-noble` but the base image is Jammy. Not sure it will work. Have you tried already?

In addition, to keep the update automatic, would be nice for `24` being a parameter depending of the build date, ie the build will inject the right version to use so we won't need to update the docker build for the CI to be up-to-date.